sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessErrorEventArgs.cs

```diff
@@ -12,6 +12,33 @@ namespace Azure.Messaging.ServiceBus
     /// </summary>
     public sealed class ProcessErrorEventArgs : EventArgs
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ProcessErrorEventArgs" /> class.
+        /// </summary>
+        ///
+        /// <param name="exception">The exception that triggered the call to the error event handler.</param>
+        /// <param name="errorSource">The source associated with the error.</param>
+        /// <param name="fullyQualifiedNamespace">The endpoint used when this exception occurred.</param>
+        /// <param name="entityPath">The entity path used when this exception occurred.</param>
+        /// <param name="identifier">The identifier of the client entity that reported this error.</param>
+        /// <param name="cancellationToken">The processor's <see cref="System.Threading.CancellationToken"/> instance which will be cancelled
+        /// in the event that <see cref="ServiceBusProcessor.StopProcessingAsync"/> is called.</param>
+        public ProcessErrorEventArgs(
+            Exception exception,
+            ServiceBusErrorSource errorSource,
+            string fullyQualifiedNamespace,
+            string entityPath,
+            string identifier,
+            CancellationToken cancellationToken)
```

Comment:
nit: call into the other ctor, and set the identifier in the body of this one.
We should probably hide the other ctor as well with EditorBrowsableNever.